<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorPatient</class>
 <widget class="QDialog" name="DoctorPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Search</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_name_2"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Surname:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_surname_2"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_search">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Patients</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_back">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Patient info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_name"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Surname:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_surname"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Sex:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_sex">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Birthdate:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_birthdate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Diagnose:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_diagnose"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_edit">
            <property name="whatsThis">
             <string>If you want to change the diagnose, enter the new one,then press</string>
            </property>
            <property name="text">
             <string>Save diagnose</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Analysis:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Treatment</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QListView" name="listView_analysis"/>
      </item>
      <item row="3" column="1">
       <widget class="QTableView" name="tableView_treatment"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit_analysis"/>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Medicine:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_medicine"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_add_analysis">
        <property name="text">
         <string>Add analysis</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_description"/>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton_add_medicine">
        <property name="text">
         <string>Add medicine</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>label_7</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>listView_analysis</zorder>
     <zorder>label_2</zorder>
     <zorder>pushButton_add_medicine</zorder>
     <zorder>pushButton_add_analysis</zorder>
     <zorder>lineEdit_analysis</zorder>
     <zorder>tableView_treatment</zorder>
     <zorder>pushButton_edit</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
